packages/wrangler/src/r2/lifecycle.ts

```diff
@@ -0,0 +1,412 @@
+import { printWranglerBanner } from "../update-check";
+import { requireAuth } from "../user";
+import formatLabelledValues from "../utils/render-labelled-values";
+import {
+	formatActionDescription,
+	getLifecycleRules,
+	isNonNegativeNumber,
+	isValidDate,
+	putLifecycleRules,
+	tableFromLifecycleRulesResponse,
+} from "./helpers";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+import type { LifecycleRule } from "./helpers";
+
+export function ListOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("bucket", {
+			describe: "The name of the R2 bucket to list lifecycle rules for",
+			type: "string",
+			demandOption: true,
+		})
+		.option("jurisdiction", {
+			describe: "The jurisdiction where the bucket exists",
+			alias: "J",
+			requiresArg: true,
+			type: "string",
+		});
+}
+
+export async function ListHandler(
+	args: StrictYargsOptionsToInterface<typeof ListOptions>
+) {
+	await printWranglerBanner();
+	const config = readConfig(args.config, args);
+	const accountId = await requireAuth(config);
+
+	const { bucket, jurisdiction } = args;
+
+	logger.log(`Listing lifecycle rules for bucket '${bucket}'...`);
+
+	const lifecycleRules = await getLifecycleRules(
+		accountId,
+		bucket,
+		jurisdiction
+	);
+
+	if (lifecycleRules.length === 0) {
+		logger.log(`There are no lifecycle rules for bucket '${bucket}'.`);
+	} else {
+		const tableOutput = tableFromLifecycleRulesResponse(lifecycleRules);
+		logger.log(tableOutput.map((x) => formatLabelledValues(x)).join("\n\n"));
+	}
+}
+
+export function AddOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("bucket", {
+			describe: "The name of the R2 bucket to add a lifecycle rule to",
+			type: "string",
+			demandOption: true,
+		})
+		.positional("id", {
+			describe: "A unique identifier for the lifecycle rule",
+			type: "string",
+			requiresArg: true,
+		})
+		.positional("prefix", {
+			describe:
+				"Prefix condition for the lifecycle rule (leave empty for all prefixes)",
+			type: "string",
+			requiresArg: true,
+		})
+		.option("expire-days", {
+			describe: "Number of days after which objects expire",
+			type: "number",
+			requiresArg: true,
+		})
+		.option("expire-date", {
+			describe: "Date after which objects expire (YYYY-MM-DD)",
+			type: "number",
+			requiresArg: true,
+		})
+		.option("ia-transition-days", {
+			describe:
+				"Number of days after which objects transition to Infrequent Access storage",
+			type: "number",
+			requiresArg: true,
+		})
+		.option("ia-transition-date", {
+			describe:
+				"Date after which objects transition to Infrequent Access storage (YYYY-MM-DD)",
+			type: "string",
+			requiresArg: true,
+		})
+		.option("abort-multipart-days", {
+			describe:
+				"Number of days after which incomplete multipart uploads are aborted",
+			type: "number",
+			requiresArg: true,
+		})
+		.option("jurisdiction", {
+			describe: "The jurisdiction where the bucket exists",
```

Comment:
nit
```suggestion
			describe: "The jurisdiction where the R2 bucket exists",
```

Reply:
This is consistent with other jurisdiction option descriptions